Library/Homebrew/utils.rb

```diff
@@ -9,6 +9,20 @@
 
 class Tty
   class << self
+    def tick
+      # necessary for 1.8.7 unicode handling since many installs are on 1.8.7
+      @tick ||= ["2714".hex].pack("U*")
+    end
+
+    def cross
+      # necessary for 1.8.7 unicode handling since many installs are on 1.8.7
+      @cross ||= ["2718".hex].pack("U*")
+    end
```

Comment:
@UniqMartin suggested this fit more nicely in `Tty` and I think it fits better with the rest of the methods here (each of which is also a function but could be a constant).